route/settings/SAMPLE-coupled.control

```diff
@@ -13,8 +13,9 @@
 ! RUN CONTROL 
 ! --------------------------------------------
 <case_name>             CASE_NAME                  ! name of simulation
-<route_opt>             1                          ! option for routing schemes 0-> both, 1->IRF, 2->KWT otherwise error 
+<route_opt>             1                          ! option for routing schemes 0->accum 1->IRF (default) 2->KWT 3->KW 4->MC 5->DW
 <doesBasinRoute>        1                          ! basin routing options   0-> no, 1->IRF, otherwise error
+<is_flux_wm>            T                          ! switch for water abstraction/injection
```

Comment:
I'm not quite sure what "is_flux_wm" does? And it looks like right now it's not connected in right? I think both the name and comment description could be improved.

Reply:
Yes, this naming is coming from different context.  Shervan was thinking about water management (wm), but we come up with more intuitive name.

Reply:
I will rename is_flux_wm (and also other similar one) in different pull request